packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -840,6 +840,25 @@ const schema: SchemaType<"Posts"> = {
     hidden: !isLWorAF
   },
 
+  glossary: {
+    type: Array,
+    typescriptType: "GlossaryTerm[]",
+    optional: true,
+    nullable: true,
+    blackbox: true,
```

Comment:
I currently think `blackbox` is basically a legacy field that doesn't do anything except add some pointless cruft to our generated types (which would even be wrong in this case).